python/pythonizations/PyROOT_pythonizationtest.py

```diff
@@ -156,6 +156,22 @@ def test04_roodataset(self):
         data = gauss.generate(ROOT.RooArgSet(x), 10000)  # ROOT.RooDataSet
         h1d = data.createHistogram("myname", x)
 
+    def test05_rooargset_iter(self):
+        """STL sequence iterator injected in RooAbsCollection, inherited by RooArgSet"""
+
+        # ROOT-10606
+        import ROOT
+
+        ws = ROOT.RooWorkspace("workspace")
```

Comment:
```suggestion
        varA = ROOT.RooRealVar("a", "a", 0.)
        varB = ROOT.RooRealVar("b", "b", 1.)
        varSet = ROOT.RooArgSet(a, b)
        for var in varSet:
           var.Print()
```

Reply:
This is shorter for testing the iterator of RooAbsCollection (since RooArgSet is a RooAbsCollection). Need to verify, though, that this also requires the iterator.

Reply:
Thanks! I just pushed your suggestion.